core/src/executor/aggregate/state.rs

```diff
@@ -116,6 +135,17 @@ impl<'a> AggrValue {
                 let cnt_square = Value::F64(count as f64).multiply(&Value::F64(count as f64))?;
                 expr_sub.divide(&cnt_square)
             }
+            Self::Stdev {
+                sum_square,
+                sum,
+                count,
+            } => {
+                let sum_expr1 = sum_square.multiply(&Value::I64(count))?;
+                let sum_expr2 = sum.multiply(&sum)?;
+                let expr_sub = sum_expr1.subtract(&sum_expr2)?;
+                let cnt_square = Value::F64(count as f64).multiply(&Value::F64(count as f64))?;
+                expr_sub.divide(&cnt_square)?.sqrt()
+            }
```

Comment:
Looks like you can try recursive `export` to reuse Variance
```rs
Self::Stdev {                                                                                                                               [0/1387]
                    sum_square,
                    sum,
                    count,
               } => Self::export(Self::Variance {
                   sum_square,
                   sum,
                   count,
               })?
               .sqrt()
```

Reply:
@devgony That's a really good point, thanks!
Then, how about adding `variance` local function or closure inside `Self::export` and let both `Self::Variance` and `Self::Stdev` to use it?